Packs/PingIdentity/Integrations/PingOne/PingOne.yml

```
@@ -0,0 +1,226 @@
+    - contextPath: Account.Username
+      description: PingOne account username
+      type: string
+    - contextPath: Account.DisplayName
+      description: PingOne account display name
+      type: string
+    - contextPath: Account.Email
+      description: PingOne account email
+      type: string
+    - contextPath: Account.Enabled
+      description: PingOne account enabled status
+      type: boolean
+    - contextPath: Account.CreatedAt
+      description: PingOne account create date
+      type: date
+    - contextPath: Account.UpdatedAt
+      description: PingOne account updated date
+      type: date
+    description: Returns a PingOne user
+  - name: pingone-create-user
+    arguments:
+    - name: username
+      required: true
+      description: Username
+    - name: populationId
+      description: PingOne population ID where the new user will be created
+    outputs:
+    - contextPath: Account.ID
+      description: PingOne account ID
+      type: string
+    - contextPath: Account.Username
+      description: PingOne account username
+      type: string
+    - contextPath: Account.DisplayName
+      description: PingOne account display name
+      type: string
+    - contextPath: Account.Email
+      description: PingOne account email
+      type: string
+    - contextPath: Account.Enabled
+      description: PingOne account enabled status
+      type: boolean
+    - contextPath: Account.CreatedAt
+      description: PingOne account create date
+      type: date
+    - contextPath: Account.UpdatedAt
+      description: PingOne account updated date
+      type: date
+    description: 'Create a PingOne user '
+  - name: pingone-update-user
+    arguments:
+    - name: username
+      required: true
+      description: Username
+    - name: formatted
+      description: Formatted PingOne name
+    - name: given
+      description: Given name
+    - name: middle
+      description: Middle name
+    - name: family
+      description: Family name
+    - name: nickname
+      description: Nickname
+    - name: title
+      description: Title
+    - name: locale
+      description: Locale
+    - name: email
+      description: Email
+    - name: primaryPhone
+      description: Primary phone number
+    - name: mobilePhone
+      description: Mobile phone number
+    - name: streetAddress
+      description: Street address
+    - name: locality
+      description: Locality
+    - name: region
+      description: Region
+    - name: postalCode
+      description: Zip code
+    - name: countryCode
+      description: Country code
+    - name: Type
+      description: Account type
+    description: Update a PingOne user
+  - name: pingone-delete-user
+    arguments:
+    - name: username
+      description: Username
+    - name: userId
+      description: User ID
+    description: Delete a PingOne user
+  dockerimage: demisto/python3:3.9.7.24076
+  runonce: false
+  subtype: python3
+fromversion: 6.0.0
+tests:
+- PingOnePlaybook_Test
```

This is the correct format for Partner support.
```suggestion
- No tests
```